packages/template-retail-react-app/app/components/_app/index.test.js

```diff
@@ -61,15 +62,20 @@ describe('App', () => {
         buildUrl
     }
 
-    test('App component is rendered appropriately', () => {
+    test('App component is rendered appropriately', async () => {
         useMultiSite.mockImplementation(() => resultUseMultiSite)
         renderWithProviders(
             <App targetLocale={DEFAULT_LOCALE} defaultLocale={DEFAULT_LOCALE} messages={messages}>
                 <p>Any children here</p>
             </App>
         )
-        expect(screen.getByRole('main')).toBeInTheDocument()
-        expect(screen.getByText('Any children here')).toBeInTheDocument()
+        const closeButton = screen.getByLabelText('Close dnt form')
+        const user = userEvent.setup()
```

Comment:
user is already included in the renderWithProvider (see [here](https://github.com/SalesforceCommerceCloud/pwa-kit/blob/develop/packages/template-retail-react-app/app/utils/test-utils.js#L197)). You can get the user object like this, you don't need to call user.setup() 
```
const {user} = nrenderWithProviders(
            <App targetLocale={DEFAULT_LOCALE} defaultLocale={DEFAULT_LOCALE} messages={messages}>
                <p>Any children here</p>
            </App>
        )
```